packages/amplify-codegen/src/utils/updateAmplifyMeta.js

```diff
@@ -30,15 +30,15 @@ module.exports = async (context, apiDetails) => {
 
 
   const amplifyMetaFilePath = context.amplify.pathManager.getAmplifyMetaFilePath();
-  fs.write(amplifyMetaFilePath, amplifyMeta);
+  fs.write(amplifyMetaFilePath, JSON.stringify(amplifyMeta, null, 2));
```

Comment:
JSON.stringify() space set to 2 can cause the content hard to read on some editors, the CLI has it as 4 in almost all places.